client-frameworks-support/testing-utilities/README.md

```diff
@@ -1,13 +1,29 @@
 ## Luigi Testing Utilities
 
-Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based microfrontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to test. 
+Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based micro frontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to mock and assert Luigi functionality. 
 
 ### LuigiMockUtil 
-This class contains certain utility helper functions needed especially in e2e-testing with [protractor](https://www.npmjs.com/package/protractor). You can simply import this module into you project and then use an instance of it to test microfrontend functionality. 
+This class contains certain utility helper functions needed when writing [protractor](https://www.npmjs.com/package/protractor) based e2e tests. You can simply import this module into you project and then use an instance of it to test microfrontend functionality. 
 
-#### Example Usage
+### How to use the library
```

Comment:
```suggestion
### How to use the library
```

I'm not sure if "the library" means the utilities as a whole or only the LuigiMockUtil class? Because we don't mention a library before this title and it's a little confusing.

Reply:
I think it is fine, because we also use the term in the left nav of our docu as well (Framework support library).